static/templates/items/item-spell-sheet.hbs

```diff
@@ -279,12 +279,18 @@
                 <a class="item-checkbox" data-target="system.overcast.valuePerOvercast.bonus">{{localize 'Bonus'}}<i
                     class="far fa-circle"></i></a>
               {{/if}}
+            {{/if}}
+            <div class="form-group">
+              <label class="label-text">{{localize 'SHEET.OtherCost'}}</label>
+              <div class="input-box">
+                  <input class="input-text" placeholder="{{localize "SHEET.OtherCostPlaceholder"}}" data-dtype="String" type="text"
+                    name="system.overcast.valuePerOvercast.cost"
+                    value="{{system.overcast.valuePerOvercast.cost}}" />
+              </div>
+            </div>
```

Comment:
Can you hide or otherwise denote this as being a WoM specific field?

Reply:
@silentmark I'm good to merge this once resolved here

Reply:
That's the motivation ;) I will finsh all my conflicts and reported issues today night

Reply:
I hope that    {{#if (settings "useWoMOvercast")}} is enough.